tests/unit/test_cli_session.py

```diff
@@ -3,6 +3,8 @@
 from mock import patch, Mock
 
 
+@patch('records_mover.session.os')
+@patch('records_mover.session.subprocess')
```

Comment:
I was seeing different coverage numbers in CircleCI vs locally because I wasn't isolating these tests sufficiently.  Fixed!